plugins/modules/win_gpo_manage.ps1

```diff
@@ -0,0 +1,147 @@
+#!powershell
+# Copyright: (c) 2021 Sebastian Gruber ,dacoso GmbH All Rights Reserved.
+# SPDX-License-Identifier: GPL-3.0-only
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#AnsibleRequires -CSharpUtil Ansible.Basic
+
+$spec = @{
+    options  = @{
+        name     = @{ type = "list"; elements = "str" }
+        folder   = @{ type = "path"; default = "C:\GPO" }
+        mode     = @{ type = "str"; choices = "import", "query", "remove", "export";required=$true }
```

Comment:
Use `state` instead as it's a more common option in Ansible. It should have the following options

* `import` -> `present`
* `query` -> this shouldn't be here but in a separate `win_gpo_manage_info` module
* `remove` -> `absent`
* `export` -> `exported`

It should also have a default of `present` just like most of the other modules that have `state`.